probes/hasAnyWorkflows/def.yml

```diff
@@ -0,0 +1,27 @@
+# Copyright 2023 OpenSSF Scorecard Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+id: hasAnyWorkflows
+short: Checks whether a project has any workflows.
+motivation: >
+  This probe is only used to determine whether Scorecard can make conclusive scoring for Dangerous Workflow-related probes.
+implementation: >
+  The probe checks whether the project has any workflows and does not consider whether they are dangerous. The probe checks the `NumWorkflows` of the raw results.
+outcome:
+  - If a project does not have any workflows, one finding with OutcomeNegative (0) is returned.
+  - If a project has any workflows, one finding with OutcomeNegative (1) is returned.
```

Comment:
copy paste errror of outcomenegative(1), should be positive?

But see the other comment first about if we still need this probe.

Reply:
Removed probe in https://github.com/ossf/scorecard/pull/3521/commits/f1ad11a3e563eea643aa230ff8b9ec5db0489b5d